nx-X11/programs/Xserver/hw/nxagent/NXdixfonts.c

```diff
@@ -929,6 +929,7 @@ doListFontsAndAliases(ClientPtr client, LFclosurePtr c)
     for (i = 0; i < nnames; i++)
 	stringLens += (names->length[i] <= 255) ? names->length[i] : 0;
 
+    memset(&reply, 0, sizeof(xListFontsReply));
     reply.type = X_Reply;
     reply.length = (stringLens + nnames + 3) >> 2;
     reply.nFonts = nnames;
```

Comment:
Second part from `nx-X11/programs/Xserver/hw/nxagent/NXdixfonts.c` should also be backported to this file.